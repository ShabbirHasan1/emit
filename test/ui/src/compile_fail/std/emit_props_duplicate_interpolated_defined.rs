fn main() {
    emit::emit!("template {x: 42}", x);
}
