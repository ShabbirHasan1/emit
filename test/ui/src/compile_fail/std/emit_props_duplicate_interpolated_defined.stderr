error: keys that exist in the template and extra pairs can only use identifiers
 --> src/compile_fail/std/emit_props_duplicate_interpolated_defined.rs:2:37
  |
2 |     emit::emit!("template {x: 42}", x);
  |                                     ^
